resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnSchedulerBackend.scala

```diff
@@ -333,6 +333,13 @@ private[spark] abstract class YarnSchedulerBackend(
           logWarning(s"Requesting driver to remove executor $executorId for reason $reason")
           driverEndpoint.send(r)
         }
+
+      // In case of yarn Miscellaneous Process is Spark AM Container
+      // Launched for the deploy mode client
+      case processInfo @ MiscellaneousProcessInfo(_, _) =>
+        logInfo(s"Sending the Spark AM info for yarn client mode")
```

Comment:
Doesn't seem like this needs to be info, perhaps change to debug unless you have specific use case here

Reply:
made it debug